src/connect_wrap.h

```diff
@@ -3,13 +3,13 @@
 
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
-#include "env.h"
 #include "req_wrap-inl.h"
 #include "async_wrap.h"
-#include "v8.h"
 
 namespace node {
 
+class Environment;
```

Comment:
`async_wrap.h` is guaranteed to have a forward declaration of this because it uses it for the `AsyncWrap` constructor